src/lib/server/models.ts

```diff
@@ -73,13 +44,16 @@ const modelConfig = z.object({
 			})
 		)
 		.optional(),
-	endpoints: z.array(combinedEndpoint).optional(),
+	endpoints: z.array(endpointSchema).optional(),
 	parameters: z
 		.object({
 			temperature: z.number().min(0).max(1),
 			truncate: z.number().int().positive(),
```

Comment:
Not sure `truncate` should be mandatory, does not make much sense use with OAI

Reply:
Yeah I agree, I also think parameters should be defined per endpoint, not per model. That would allow for specifying the right params per endpoint type.